### PR TITLE
docs: Added Deploying LLMs into production + a new ecosystem

### PR DESCRIPTION
**Pull request by kouroshHakha**

### CODE REVIEW

**Comment by skcoirz** [comment]
ah, this is a good idea. Conventionally, inference services are hosted by LLM providers. I was thinking of LangChain as a part of product logic, but after reading your PR, I realized we are a valuable inference host too especially after so many customization through multi-api combination, (prefix) prompt engineering and complex agent features.

Thank you for sharing with us your great ideas!

**Review by hwchase17 — COMMENTED** [review]
this is more of a ray integration than a comprehensive guide on deploying LLMs in production

this seems more suitable to go in the ecosystem section

**Comment by kouroshHakha** [comment]
Hey @hwchase17 , thanks for your input. I'm considering revising the section on "Deployment of LLMs in production." Instead of the current story about how Ray serve can assist with deployment, I'm thinking of creating a more general tutorial on the key concepts to consider when deploying LLMs (autoscaling, spot-instance serving, defining end-points, etc). Then I can link it to the ray integration page for follow-ups. What do you think of this approach? Do you have any other suggestions or ideas to improve this section?

**Comment by richardliaw** [comment]
Hey there - also from Ray team here -- your feedback makes sense @hwchase17 !

Probably better for this PR to actually start the "comprehensive guide for deploying LLMs in production". We could use both the Ray and the BentoML (https://github.com/ssheng/BentoChain) example as a starting point, so that it doesn't just look like a basic Ray integration. 

And the sections of the guide would include the parts that @kouroshHakha mentioned.

Thoughts?

**Comment by kamil-kaczmarek** [comment]
I think it makes sense. we can keep the _comprehensive guide for deploying LLMs in production_ focused more on concept understanding and provide solution starters. From there we can link to more detailed examples.

**Comment by kouroshHakha** [comment]
Hey @hwchase17, I updated the main section to be more general and more in the lines of a _comprehensive guide for deploying LLMs in production_. I am thinking of linking different serving ecosystems (ray serve, bentochain, etc) to this main doc and this should live somewhere in the main doc pages that is super visible (maybe under LLMs?) I can polish the text / figures a bit more if the outline sounds reasonable to you. Thanks.

**Review by hwchase17 — COMMENTED** [review]
ray serve notebook looks good!

Deploying LLMs notebook is also good in content, but i think in wrong place. `modules` is very specific to code in the library, this is more of (very good) general purpose documentation. 

i would suggest we move this to the additional resources section (and maybe make it a markdown file, no reason for it to be ipynb)

File: docs/additional_resources/deployments.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ This section covers several options for that. Note that these options are meant
 
 What follows is a list of template GitHub repositories designed to be easily forked and modified to use your chain. This list is far from exhaustive, and we are EXTREMELY open to contributions here.
 
+## [Anyscale](https://www.anyscale.com/model-serving)
+
+Anyscale is a unified compute platform that makes it easy to develop, deploy, and manage scalable LLM applications in production using Ray.
+With anyscale you can scale the most challenging LLM-based workloads and both develop and deploy LLM-based apps on a single compute platform.
```
Comment (kamil-kaczmarek) [inline docs/additional_resources/deployments.md:12]:
```suggestion
With Anyscale you can scale the most challenging LLM-based workloads and both develop and deploy LLM-based apps on a single compute platform.
```

**Review by kamil-kaczmarek — APPROVED** [review]

**Comment by kamil-kaczmarek** [comment]
Hi @hwchase17 thanks for the suggestions.

@kouroshHakha implemented requested the changes. Please have a look.

**Comment by kouroshHakha** [comment]
@hwchase17 I think this PR is ready to be merged. Can someone from your team do a final pass? Thanks.

**Review by hwchase17 — APPROVED** [review]
lgtm - thanks! sorry for dealy

**Pull request merged**